### ISSUE TITLE
Data portal - Depth filter not working as expected when depth is specified as a range in mongo.

### ISSUE THREAD

**Issue by aclum** [microbiomedata/nmdc-server#1080]
I had the NEON benthic samples selected in data portal dev (study id nmdc:sty-11-pzmd0x14) and noticed that the depth 'in between' filter is not working properly. For all of these samples in mongo the values for depth.has_maximum_numeric_value=1 and depth.has_minimum_numeric_value=0. I was expecting depth filter of 'in between' 0.1 - 0.5 to return all samples but it returns none. When i run a similar search in data portal prod using biosamples that are part_of nmdc:sty-11-28tm5d36 which also has depth as a range I encounter the same behavior.

I would expect the screenshot below to return 560 samples since all benthic samples (study id nmdc:sty-11-pzmd0x14) were sampled between 0-1 meters.

env: dev and prod data portal

![Screenshot 2023-11-06 at 2 59 25 PM](https://github.com/microbiomedata/nmdc-server/assets/8196591/5e5ef251-f1c7-41b8-87c3-4664be80625f)

**Comment by aclum** [comment]
@pkalita-lbl took a quick look at the ingest code and it doesn't appear that the ingest codes knows that depth can be a range in the data in mongo. Unclear if that issue needs to be a separate ticket from this.

**Comment by naglepuff** [comment]
Nope, that's the same issue. I believe the ingest code brings in the minimum depth. It's unclear without digging in what it would take to support overlapping range queries in general. A more specific fix could be added to address depth in particular.

**Comment by aclum** [comment]
Based on infra meeting for now  if faceted search uses 'in between' unless the range in the records is full contained by the search values results shouldn't be returned.
sample A has range of .5-1m and search is 'in between' 0-.75m -> do not show sample A.

**Comment by ssarrafan** [comment]
@marySalvi @aclum is this active?  Will it be worked on next sprint?

**Comment by ssarrafan** [comment]
No updates. Doesn't appear to be active. Removing from sprint. Adding unfinished/backlog labels. FYI @aclum @marySalvi

**Comment by ssarrafan** [comment]
@aclum Mary can work on this if it's high priority you can add to next sprint.
